patches/0007-Add-support-for-QMJ5.patch

```diff
@@ -0,0 +1,486 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Ennui Langeweile <85590273+EnnuiL@users.noreply.github.com>
+Date: Mon, 3 Jul 2023 22:01:25 -0300
+Subject: [PATCH] Add support for QMJ5
+
+
+diff --git a/build.gradle b/build.gradle
+index d9e90d46948d9335225f959e865af681056a3d51..eb546b11c1c9e10e083ffc0e4358fb1f8a7b8d73 100644
+--- a/build.gradle
++++ b/build.gradle
+@@ -107,8 +107,11 @@ dependencies {
+ 	// Kapt integration
+ 	compileOnly('org.jetbrains.kotlin:kotlin-gradle-plugin:1.8.0') // Must match the version included with gradle.
+ 
++	// Dependencies we import
++	implementation('org.quiltmc:quilt-json5:1.0.4+final')
```

Comment:
Use https://github.com/quiltmc/quilt-parsers instead (`org.quiltmc.parsers.json` and `org.quiltmc.parsers.gson`, it includes its own gson adapter and some bugfixes.

Reply:
ohhh, so that's what QUP had become! I might end not being able to tackle this today, but I'll see if I can at least tackle it until tomorrow

Reply:
Addressed! I appreciate the file savings :p